src/os/dispatch/dispatcher.js

```diff
@@ -0,0 +1,33 @@
+goog.module('os.dispatch.dispatcher');
+goog.module.declareLegacyNamespace();
+
+const EventTarget = goog.require('goog.events.EventTarget');
+
+/**
+ * @type {EventTarget|null}
+ */
+let instance = null;
+
+/**
+ * singleton
+ * @return {EventTarget}
```

Comment:
`!EventTarget` would be more accurate, given the return type is guaranteed to be non-null.